documentation/ci-fix.md

```diff
@@ -100,22 +100,7 @@ The [LintDiff validation tool](https://github.com/Azure/azure-openapi-validator)
 Refer to [openapi-authoring-automated-guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md) for detailed description of all lint rules and how-to-fix guidance.  
 If that guidance is not enough, please also refer to the [LintDiff rules.md doc](https://github.com/Azure/azure-openapi-validator/blob/main/docs/rules.md). It links to `.md` files related to given error, containing instructions how to fix them.
 
-### Run linter locally:
-
-#### Prerequisites:
-npm install -g autorest
-
-#### Given a swagger spec, run linter:
-```
-autorest --v3 --spectral --validation --azure-validator --use=@microsoft.azure/openapi-validator@latest --input-file=<path-to-spec>
-```
-#### Given a readme file, run linter:
-```
-autorest --v3 --spectral --validation --azure-validator --use=@microsoft.azure/openapi-validator@latest [--tag=<readme tag>] <path-to-readme>
-```
-
-Please see [readme](https://github.com/Azure/azure-openapi-validator/blob/main/README.md) for how to install or run tool in details.
-Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/linter).
+To reproduce LintDiff failures locally, see [CONTRIBUTING.md / How to run LintDiff locally](https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-run-lintdiff-locally).
```

Comment:
Should we instead link to https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#execute-your-local-lintdiff-code? Most of the setup steps are not applicable when trying to run inside of the specs repo context (aside from the install autorest step), they are instead indented for running in that repo. We should try and keep these instructions as clear as possible for what folks in the spec repo context need to do.

Reply:
Thanks, very good point. I have overhauled the target doc to make the instructions easier to follow:

The new doc:
- https://github.com/Azure/azure-openapi-validator/blob/f19e91fc9f78eebffeba50be319abfa0dfccc1ab/CONTRIBUTING.md#how-to-locally-reproduce-a-lintdiff-failure-occurring-on-a-pr

The PR:
- https://github.com/Azure/azure-openapi-validator/pull/539